geonode/mngmt_commands_http/management/commands/ping_mngmt_commands_http.py

```diff
@@ -0,0 +1,42 @@
+#########################################################################
+#
+# Copyright (C) 2021 OSGeo
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+#########################################################################
+from time import sleep
+from django.core.management.base import BaseCommand
+from geonode.mngmt_commands_http.decorators import expose_command_over_http
+
+
+@expose_command_over_http
+class Command(BaseCommand):
```

Comment:
This looks like more a "test" command than a real one. I'd consider moving this one in the tests package then keeping it as an exposed command. This one is very useful in a "smoke test"

Reply:
@mattiagiupponi I didn't get the idea here.
* Do you mean move it to another app or inside a folder `tests` of this app?
* Django does not support management commands inside the `tests` folder, so we would not be able to use or expose it.

Reply:
Yes, the idea was to move it under the test package, but if Django does not support it just keep it as is